internal/plugin/server.go

```diff
@@ -151,7 +151,9 @@ func (plugin *NvidiaDevicePlugin) waitForMPSDaemon() error {
 	}
 	// TODO: Check the started file here.
 	// TODO: Have some retry strategy here.
-	if err := mps.NewDaemon(plugin.rm).AssertHealthy(); err != nil {
+	// TODO: /mps here is represents the path relative to the /driver-root.
```

Comment:
This comment is not correct now that you have rebased this branch of https://github.com/NVIDIA/k8s-device-plugin/pull/516, right?

Also, we are hardcoding `/mps` here since that is always the containerPath, correct?

Reply:
Yes. This would need a rebase on #516. I will update the comment then.